test/app/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+yarnPath: ../../.yarn/releases/yarn-3.5.1.cjs
```

Comment:
❓ We're using `pnp` for the test app ?

Reply:
Hum somehow the `nodeLinker` value is inherited from the root `package.json`

```
$ cd test/app
$ yarn config --why | grep nodeLinke
➤ YN0000: nodeLinker                    /Users/benoit.zugmeyer/go/src/github.com/DataDog/browser-sdk/.yarnrc.yml            'node-modules'
```

Reply:
I removed `test/app/.yarnrc.yml` as it turns out everything is inherited correctly

Reply:
👍